omou-frontend/src/reducers/initialState.js

```diff
@@ -186,6 +186,246 @@ export default {
+                capacity: 12,
+                filled: 12,
+                grade: 8,
+                description: "This course is to prepare 5th grade students to pass the 6th grade math placement test.",
+                room_id: 2,
+            },
+            7: {
+                course_id: 7,
+                course_title: "SAT 2 Math 2",
+                schedule: {
+                    start_date: "03-08-2019",
+                    end_date: "05-08-2019",
+                    start_time: "T17:00",
+                    end_time: "T20:00",
+                    days: [5],
+                },
+                instructor_id: 2,
+                tuition: 700,
+                capacity: 15,
+                filled: 14,
+                grade: 10,
+                description: "This course is to prepare 5th grade students to pass the 6th grade math placement test.",
+                room_id: 2,
+            },
+            8: {
+                course_id: 8,
+                course_title: "AP Chemistry",
+                schedule: {
+                    start_date: "02-23-2019",
+                    end_date: "04-27-2019",
+                    start_time: "T09:00",
+                    end_time: "T12:00",
+                    days: [6],
+                },
+                instructor_id: 3,
+                tuition: 700,
+                capacity: 15,
+                filled: 8,
+                grade: 10,
+                description: "This course is to prepare 5th grade students to pass the 6th grade math placement test.",
+                room_id: 2,
+            },
+            9: {
+                course_id: 9,
+                course_title: "AP Calc B/C",
+                schedule: {
+                    start_date: "03-08-2018",
+                    end_date: "05-12-2019",
+                    start_time: "T17:30",
+                    end_time: "T10:30",
+                    days: [5],
+                },
+                instructor_id: 2,
+                tuition: 700,
+                capacity: 15,
+                filled: 13,
+                grade: 10,
+                description: "This course is to prepare 5th grade students to pass the 6th grade math placement test.",
+                room_id: 2,
+            },
+            10: {
+                course_id: 10,
+                course_title: "AP Calc A/B",
+                schedule: {
+                    start_date: "03-09-2019",
+                    end_date: "05-11-2019",
+                    start_time: "T13:00",
+                    end_time: "T16:00",
+                    days: [6],
+                },
+                instructor_id: 1,
+                tuition: 700,
+                capacity: 15,
+                filled: 2,
+                grade: 10,
+                description: "This course is to prepare 5th grade students to pass the 6th grade math placement test.",
+                room_id: 2,
+            }
+        },
+        "CourseSessions": {
```

Comment:
Based off of this, only 2 courses have any sessions at all (course id 1 and course id 2). You need to make more fake data for the other 8 courses. Also spend some time creating at least 15 sessions per course since a typical course is 15 weeks (15 weeks/semester)